ledger/participant-integration-api/src/main/scala/db/migration/translation/ContractSerializer.scala

```diff
@@ -38,9 +37,8 @@ private[migration] object ContractSerializer extends ContractSerializer {
       TransactionCoder
         .decodeVersionedContractInstance[ContractId](
           ValueCoder.CidDecoder,
-          TransactionOuterClass.ContractInstance.parseFrom(
-            Reader.damlLfCodedInputStream(stream)
-          ),
+          TransactionOuterClass.ContractInstance
+            .parseFrom(com.google.protobuf.CodedInputStream.newInstance(stream)),
```

Comment:
Note that this class is only used by some old database migrations scripts. A change here will likely not be covered by any tests.